src/main/java/com/epam/prd/jjop/helpers/unmodifiablelist_vs_list_of/Main.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.prd.jjop.helpers.unmodifiablelist_vs_list_of;
```

Comment:
By convention, packages are named without underscores. All words should be concatenated. I know that `unmodifiablelistvslistof` doesn't looks but regarding to the convention is should be like this.